css/style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Chicle&family=Poppins:wght@300;400;500;600;700;800;900&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,900&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.MainContainer {
+  width: 100%;
+  height: 100vh;
+}
+
+header {
```

Comment:
- [ ] [Optional] This is optional but for best practices, I suggest you use classes instead of tags to style your page